Challenge IV/scriptIV.js

```diff
@@ -0,0 +1,8 @@
+$("#button").click(function () {
+	alert("Your data is being processed... please wait while we file your information.");
+	const name = $("#name").val();
+	const favoriteFood = $("#favoriteFood").val();
+	const favoriteColor = $("#favoriteColor").val();
+	console.log(name, favoriteFood, favoriteColor);
+	$("table").append("<tr><td>"+name+"</td><td>"+favoriteFood+"</td><td>"+favoriteColor+"</td></tr>")
+});
```

Comment:
Excellent job using ES6 format (`const`). 

In line 7 you can use ES6 string interpolation to help make it a bit more readable. It would look like this instead:

``` javascript
$("table").append(`<tr><td>${name}</td><td>${favoriteFood}</td><td>${favoriteColor}</td></tr>`)
``` 

Using this format, you can just write HTML the way you would in your HTML file, and whenever you reference a variable name, you just need to include it in curly brackets with a dollar sign `${variableName}`. You also will need to include everything inside of backticks (which are near the number 1 on your keyboard.)